android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -163,17 +164,32 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
 
     @ReactMethod
     override fun submitFingerprintAcceptAuthenticationRequest(promise: Promise) {
-        oneginiSDK.fingerprintAuthenticationRequestHandler?.acceptAuthenticationRequest()
+        if (oneginiSDK.fingerprintAuthenticationRequestHandler == null) {
```

Comment:
There is a Kotlin way how to deal with nullability. 
```
oneginiSDK.fingerprintAuthenticationRequestHandler?.let { fingerprintHandler ->
            fingerprintHandler.acceptAuthenticationRequest()
            promise.resolve(null)
        } ?: promise.reject(OneginiWrapperErrors.FINGERPRINT_IS_NOT_ENABLED.code, OneginiWrapperErrors.FINGERPRINT_IS_NOT_ENABLED.message)
```
more info on kotlin scope functions -> https://kotlinlang.org/docs/scope-functions.html#let

This way there's no need for force unwrap. Please copy the pattern for the rest of the methods in this file.